specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/mysql.json

```diff
@@ -1321,6 +1517,115 @@
         "modelAsString": false
       }
     },
+    "MinimalTlsVersion": {
+      "type": "string",
+      "description": "Enforce a minimal Tls version for the server.",
+      "enum": [
+        "TLS1_0",
+        "TLS1_1",
+        "TLS1_2",
+        "TLSEnforcementDisabled"
+      ],
+      "x-ms-enum": {
+        "name": "MinimalTlsVersionEnum",
+        "modelAsString": true
+      }
+    },
+    "ServerPrivateEndpointConnection": {
+      "description": "A private endpoint connection under a server",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/ServerPrivateEndpointConnectionProperties",
```

Comment:
we want these flattened don't we?

Reply:
This is dictated by the network team. No, we do not want it flattened.